modules/ossm-rn-fixed-issues.adoc

```diff
@@ -49,6 +49,15 @@ Namespace starting with `kube` is hidden from Kiali.
 
```

Comment:
I didn't see a content type at the beginning of this file.

Reply:
Add to both files.